src/components/Header/SearchBar/SearchBar.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <form class="search-bar group flex flex-row" @submit.prevent="handleSearch">
+    <InputField
+      v-model="text"
+      v-bind="$attrs"
+      class="flex-grow input-field"
+      :connection-sides="['end']"
+      input-id="search-bar"
+      type="search"
+      name="q"
+    >
+      <!-- @slot Extra information such as loading message or result count goes here. -->
+      <slot />
+    </InputField>
+    <SearchButton type="submit" />
+  </form>
+</template>
+
+<script>
+import { computed } from '@nuxtjs/composition-api'
+
+import InputField from '~/components/InputField/InputField.vue'
+import SearchButton from '~/components/Header/SearchBar/SearchButton.vue'
+
+/**
+ * Displays a text field for a search query and is attached to an action button
+ * that fires a search request. The loading state and number of hits are also
+ * displayed in the bar itself.
+ */
+export default {
+  name: 'SearchBar',
+  components: {
+    InputField,
+    SearchButton,
+  },
+  inheritAttrs: false,
+  model: {
+    prop: 'value',
+    event: 'input',
+  },
+  props: {
+    /**
+     * the search query given as input to the field
+     */
+    value: {
+      type: String,
+      default: '',
+    },
+  },
+  setup(props, { emit }) {
+    const text = computed({
+      get() {
+        return props.value
+      },
+      set(value) {
+        emit('input', value)
+      },
+    })
+
+    const handleSearch = () => {
+      emit('submit')
+    }
+
+    return {
+      text,
+
+      handleSearch,
+    }
+  },
+}
+</script>
+
+<style>
+/* Removes the cross icon to clear the field */
+input[type='search']::-webkit-search-decoration,
+input[type='search']::-webkit-search-cancel-button,
+input[type='search']::-webkit-search-results-button,
+input[type='search']::-webkit-search-results-decoration {
+  -webkit-appearance: none;
+}
```

Comment:
Why remove this? It's especially useful on mobile.

cc @panchovm 

Also the way these styles are applied means they will be applied globally. We should use either `scoped` or `module` and associate these modifications to a specific class at the very least.

Reply:
This can't be scoped because the input-field being styled is in a different component. But made the selector more "selective" in a7d1e98.

Reply:
That's fine, I still think we should make sure removing it is the right choice. Why are we removing it? The button is especially useful on mobile devices where selecting the contents of an input isn't necessarily easy across devices.

Reply:
I agree with the utility of the cross on mobiles. It's good from an a11y perspective. My main complaint against it was that it also appears on desktops, where its utility is doubtful. I'll let @panchovm take the call on this one.

Reply:
Sorry I am confused, what are we removing?

Reply:
Now I understand, sorry for the confusion. The clear action never was in the redesign, but you are right; it adds so much value to the search experience. I will work on adding it to the Design Library's components.